### PR TITLE
Automated Test - LRQA-67513 | master

### PR DESCRIPTION
**Pull request by 91nilo**

### CODE REVIEW

**Comment by 91nilo** [comment]
https://issues.liferay.com/browse/LRQA-67513, cc: @rodrigocunhaa

**Comment by 91nilo** [comment]
Fixed conflicts and added code from missing steps.

**Comment by 91nilo** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase [hunk 1]
```diff
@@ -276,12 +276,104 @@ definition {
...
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase:304]: For this Blogs macro could you please change it to 
`JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");`

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase [hunk 2]
```diff
@@ -276,12 +276,104 @@ definition {
...
+			entrySubtitle = "Subtitle",
+			entryTitle = "Blogs Entry A");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase:309]: For this Blogs macro could you please change it to 
`JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");`

**Comment by 91nilo** [comment]
@CalumR23 Fixed the macro. Thank you!

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
Forwared to BChan [Pull Request](https://github.com/CalumR23/liferay-portal/pull/6)

**Pull request closed without merging**